### PR TITLE
[RW-9847] Update AoU docker images to 2.1.20

### PR DESCRIPTION
**Pull request by yonghaoy**
This PR upgreade AoU docker image and fixed breaking test. 
The newer AoU notebook also upgrade notebook and kernel spec is different now. See screenshot below, Python3 -> Python3(ipykernel). 

Screenshot 2023-05-16 at 1 29 23 PM

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by yonghaoy** [comment]
Jenkins retest

**Comment by jdcanas** [comment]
jenkins retest

**Comment by yonghaoy** [comment]
Jenkins retest

**Comment by jdcanas** [comment]
jenkins retest

**Comment by jdcanas** [comment]
jenkins retest

**Comment by jdcanas** [comment]
jenkins retest

**Comment by yonghaoy** [comment]
jenkins retest

**Comment by yonghaoy** [comment]
jenkins retest

**Comment by yonghaoy** [comment]
jenkins retest

**Comment by LizBaldo** [comment]
jenkins retest

File: jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ terra_jupyter_r="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:2.1.7"
 terra_jupyter_bioconductor="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:2.1.8"
 terra_jupyter_hail="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-hail:1.0.24"
 terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.2.9"
-terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.1.17"
```
Comment (tarekmamdouh) [inline jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh:25]: please also update terra_jupyter_aou_old values

Comment (yonghaoy) [reply jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh:25]: Old value should match to the value we are currently using: https://github.com/all-of-us/workbench/blob/main/api/config/config_prod.json#L13
seems 2.1.15 is still the right value.

**Comment by yonghaoy** [comment]
tested locally and I can see snippets menu now

**Review by tarekmamdouh — APPROVED** [review]
LGTM

**Comment by yonghaoy** [comment]
jenkins retest

**Review by jdcanas — APPROVED** [review]

**Comment by yonghaoy** [comment]
jenkins retest

**Comment by yonghaoy** [comment]
jenkins retest

**Comment by yonghaoy** [comment]
jenkins retest

**Comment by yonghaoy** [comment]
jenkins retest

**Comment by yonghaoy** [comment]
jenkins retest

**Comment by yonghaoy** [comment]
jenkins retest

**Comment by yonghaoy** [comment]
jenkins retest

**Comment by yonghaoy** [comment]
jenkins retest

**Pull request merged**